sn_transfers/src/client_transfers/mod.rs

```diff
@@ -55,7 +55,7 @@ pub struct Inputs {
 /// The created dbcs and change dbc from a transfer
 /// of tokens from one or more dbcs, into one or more new dbcs.
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
-pub struct Outputs {
+pub struct TransactionOutputs {
```

Comment:
This here is not a trivial one, it comes back and bites us all the time.

I think though.... that perhaps `TransferOutputs` would be better for this one, considering the names in other parts, for consistency.

I've tried defining the difference, in some comment somewhere.
So far, the simplest convention I think is to only call the type `DbcTransaction` a `transaction`, and everything else, which is more general referring to the operations and steps, `transfer`.

Reply:
Agreed, good one!